hasher-matcher-actioner/hmalib/lambdas/api/submit.py

```diff
@@ -97,6 +99,18 @@ def get_content_ref_details(self) -> t.Tuple[str, ContentRefType]:
         return (self.content_id, ContentRefType.DEFAULT_S3_BUCKET)
 
 
+@dataclass
+class SubmitContentHashRequestBody(SubmitRequestBodyBase):
+    signal_value: str
+    signal_type: str  # SignalType.getname() values
+    content_url: t.Optional[str]
+
+    def get_content_ref_details(self) -> t.Tuple[str, ContentRefType]:
+        if self.content_url:
+            return (self.content_url, ContentRefType.URL)
+        return ("", ContentRefType.NONE)
```

Comment:
`return (self.content_url or "", ContentRefType.NONE)`?

Reply:
? that is not the same behavior as here as far as I can tell